tests/components/google_assistant/test_google_assistant.py

```diff
@@ -415,13 +362,14 @@ def test_execute_request(hass_fixture, assistant_client):
     body = yield from result.json()
     assert body.get('requestId') == reqid
     commands = body['payload']['commands']
-    assert len(commands) == 8
+    assert len(commands) == 6
```

Comment:
New implementation will group all successful commands for 1 device in a single result object.